client/signup/steps/sell-details/sell-details.scss

```diff
@@ -0,0 +1,76 @@
+@import '@wordpress/base-styles/_breakpoints.scss';
+@import '@wordpress/base-styles/_mixins.scss';
+
+.sell-details__form {
+	margin-top: 12px;
+}
+
+.sell-details__form-fieldset {
+	$gray-60: #50575e;
+	$gray-40: #787c82;
+
+	margin-bottom: 24px;
+
+	.form-label {
+		margin-bottom: 8px;
+		font-weight: 500; /* stylelint-disable-line scales/font-weights */
+		color: $gray-60;
+		border-radius: 4px; /* stylelint-disable-line scales/radii */
+
+		.form-label__optional {
+			font-size: inherit;
+			color: $gray-40;
+			text-transform: lowercase;
+
+			&::before {
+				content: '(';
+			}
+
+			&::after {
+				content: ')';
+			}
+		}
+
+		@include break-small {
+			font-size: $font-body;
+		}
+	}
+
+	input.form-text-input {
+		height: 44px;
+		line-height: 44px;
+		border-radius: 4px; /* stylelint-disable-line scales/radii */
+
+		&:focus,
+		&:focus:hover {
+			border-color: #646970;
+			box-shadow: 0 0 0 2px #e2eaf1;
+		}
+	}
+
+	.form-setting-explanation {
+		display: flex;
+		align-items: center;
+		margin: 16px 0;
+		font-style: normal;
+		color: $gray-40;
+
+		.sell-details__form-icon {
+			margin-right: 8px;
+		}
+	}
+}
+
+.button.sell-details__submit-button.is-primary {
+	padding: 9px 48px;
+	border-radius: 4px; /* stylelint-disable-line scales/radii */
+	font-weight: 500; /* stylelint-disable-line scales/font-weights */
+
+	// override unnecessary super specificity added by another class
```

Comment:
Aren't we the ones introducing the `.sell-details__submit-button` class? I'm not sure how this could be unavoidable. 🤔

Reply:
I've kept it from the site-options component to avoid breaking anything, but it seems it doesn't really override anything even on the original component... 

Anyway, I've refactored this step and this file doesn't exist anymore :)

Reply:
Revolving the conversation as the file doesn't exist anymore